sw/device/tests/meson.build

```diff
@@ -1164,12 +1164,12 @@ sw_tests += {
 }
 
 # PWRMGR all reset test.
```

Comment:
Meson is going away. I'd take this as an opportunity to delete the rule instead. :)